uber/tests/models/test_attendee.py

```diff
@@ -241,7 +241,7 @@ def test_badge_at_con(self, monkeypatch, at_con):
         monkeypatch.setattr(Attendee, 'is_new', False)
         a = Attendee(badge_num=1)
         a._misc_adjustments()
-        assert not a.checked_in
+        assert a.checked_in
```

Comment:
@kitsuta fixed this elsewhere, including it. verified this is indeed what we want.